Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Docker demo image, as used on try.jupyter.org and tmpnb.org
 
-FROM jupyter/minimal-notebook:4.0
+FROM jupyter/minimal-notebook:2d878db5cbff
```

Comment:
We should probably go to a much newer version of the stacks if we're just going to skip ruby and move for now:

https://github.com/jupyter/docker-stacks/wiki/Docker-build-history

Latest there is 4.1. The Notebook 4.2 PR is still pending on stacks waiting for ipywidgets in conda. (Haven't checked in a few days. Will in a minute.)